instrumentation/resources/library/src/main/java9/io/opentelemetry/instrumentation/resources/ProcessArguments.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.resources;
+
+final class ProcessArguments {
+
+  static String[] getProcessArguments() {
+    return ProcessHandle.current().info().arguments().orElseGet(() -> new String[0]);
+  }
```

Comment:
Leveraging the module system here is sooooooo much nicer than the reflection in the original version. ❤️